mkdocs.yml

```diff
@@ -82,7 +82,7 @@ validation:
   absolute_links: ignore
 
 extra_javascript:
-  - https://polyfill.io/v3/polyfill.min.js?features=es6
+  - https://cdnjs.cloudflare.com/polyfill/
```

Comment:
https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6

Reply:
Thank you for this!